main.go

```diff
@@ -110,9 +97,14 @@ func run(ctx context.Context) {
 		fmt.Printf("Error initializing logger: %v", err)
 		os.Exit(1)
 	}
-	packages.DebugLogger = log.New(&logWriter{}, "", 0)
+	ctx = clog.WithLabels(ctx, map[string]string{"instance_name": config.Name()})
```

Comment:
Isn't this already included in the log event since it comes from the instance?

Reply:
Thats just instance id unfortunately.